File: array-string/remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.go
func removeDuplicates(nums []int) int {
    if (len(nums) == 1) {
        return 1
    }

    pos := 0;
    for i := 1; i < len(nums); i++ {
        if (nums[pos] != nums[i]){
            pos++;
            nums[pos] = nums[i]
        }
    }

    return pos + 1
};

File: array-string/remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cs
public class Solution
{
    public int RemoveDuplicates(int[] nums)
    {
        if (nums.Length == 1)
            return 1;

        int k = 0;
        for (int i = 1; i < nums.Length; i++)
        {
            if (nums[k] != nums[i])
            {
                k++;
                nums[k] = nums[i];
            }
        }
        k++;

        return k;
    }
}